common/components/flag/macro.njk

```diff
@@ -0,0 +1,3 @@
+{% macro appFlag(params) %}
```

Comment:
What's the difference between this component and the "tag" component? It feels like it would be better to adapt that component to support this variation rather than introduce another component with a similar name.

Reply:
I went to the components library to see what the actual difference was, and it looks like there's a rendering error - https://booksecuremove-pr-1096.herokuapp.com/components/flag - I dont know if this is in the component, or the library route

Reply:
Good spot! It looks like there's some form of position relative missing?

Reply:
In my mind if feels more like a class modifier that can be applied to `app-tag` to produce the different visual style rather than a separate component.

Reply:
Or maybe even an attribute on the component rather than a class. It feels very similar to how the GOV.UK Design System handles the [Start button variation](https://design-system.service.gov.uk/components/button/#start-buttons) with the `isStartButton` option.

Reply:
There was quite a bit of difference. It was a flag to put some content inside the tag, not the tag itself. Anyhow, the `tag` component now has a `flag` attribute and the `flag` component is gone.